lib/mcb/commands/courses/sync_to_find.rb

```diff
@@ -23,11 +23,6 @@
     raise CriExitException.new(is_error: true)
   end
 
-  courses.each do |course|
-    ManageCoursesAPIService::Request.sync_course_with_search_and_compare(
```

Comment:
Shouldn't that file be removed as well?

app/services/manage_courses_api_service.rb